quickstart/202-machine-learning-moderately-secure-existing-VNet/variables.tf

```diff
@@ -0,0 +1,69 @@
+variable "name" {
+  type        = string
+  description = "Name of the deployment"
+}
+
+variable "environment" {
+  type        = string
+  description = "Name of the environment"
+  default     = "dev"
+}
+
+variable "location" {
```

Comment:
There should be a default value here to be consistent with the other examples. I believe they used "East US" as a default.

Reply:
good spot!